maat-court-data-api/src/main/java/gov/uk/courtdata/dces/controller/ConcorContributionsRestController.java

```diff
@@ -34,11 +35,11 @@ public class ConcorContributionsRestController {
     @StandardApiResponse
     @GetMapping(value = "/concor-contribution-files", produces = MediaType.APPLICATION_JSON_VALUE)
     @Operation(description = "Get a list of Contributions files")
-    public ResponseEntity<List<String>> getConcorContributionFiles(@RequestParam(name = "status") final ConcorContributionStatus status) {
+    public ResponseEntity<List<ConcorContributionResponse>> getConcorContributionFiles(@RequestParam(name = "status") final ConcorContributionStatus status) {
         log.info("Get Concor contribution files with status {}" ,status);
-        final List<String> concorFiles = concorContributionsService.getConcorFiles(status);
-        log.info("findContributionFiles count {}", concorFiles.size());
-        return ResponseEntity.ok(concorFiles);
+        final List<ConcorContributionResponse> contributionResponses = concorContributionsService.getConcorContributionFiles(status);
```

Comment:
I don't see what benefit the `final` modifier provides here? The method is very short (4 lines!) and readable :)
Marking `contributionResponses` as final seems a bit superfluous.

Also the Developer Guidelines as they currently stand do not recommend using the final modifier for local variables.

> Use the final keyword judiciously. Only use final:
> On constants, fields and parameters to indicate immutability
> On methods and classes which absolutely must not be overloaded

https://dsdmoj.atlassian.net/wiki/spaces/ASLST/pages/3896049821/Developer+Guidelines